<launch>

	<node pkg="robot" type="numbers_detect.py" name="numbers_detect" output="screen">
	</node>

</launch>